blog(embertimes135): EmberMap Native Decorator (#525)
ember-learn/ember-blog: source/2020-02-14-the-ember-times-issue-135.md

<SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹

<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
Learn about Native Decorator Support from EmberMap 🐹🗺️,
READMORE

---

## [EmberMap: Learn About Native Decorator Support 🐹🗺️](https://twitter.com/ember_map/status/1225464396471787525)

> Check out the latest EmberMap episode of "What's New in Ember" which talks about what the [Native Decorator Support](https://emberjs.github.io/rfcs/0440-decorator-support.html) feature from Ember 3.10 means for you and the overall idea why [Decorators](https://emberjs.github.io/rfcs/0408-decorators.html) are so important for Ember's adoption of ES Classes.

Reviewer: I removed a few words for brevity, if that's all right.
```suggestion
Check out the latest EmberMap episode of "What's New in Ember." It talks about what the [Native Decorator Support](https://emberjs.github.io/rfcs/0440-decorator-support.html) feature from Ember 3.10 means for you and why [Decorators](https://emberjs.github.io/rfcs/0408-decorators.html) are important for Ember's adoption of ES Classes.
```

> With Ember 3.10, all the core APIs and surrounding ecosystem libraries can start adopting decorators as per the [stage 1 implementation](https://github.com/tc39/proposal-decorators) which would pave the way for ES classes in Ember. However, the happy path is still to wait until [the Octane release or 3.15](https://blog.emberjs.com/2019/12/20/ember-3-15-released.html) to start using both ES classes and decorators together for all of your Ember app code.

Reviewer: I haven't watched the video yet, so in this paragraph, I was a little confused by what you had meant to convey regarding Ember 3.10 versus 3.15.

Author: they talked about the history – im gonna submit for now since ill be away from laptop lol but feel free to rewrite as needed :) thanks for the review!
